test/model/modelInfo/ModelInfo.test.ts

```diff
@@ -0,0 +1,148 @@
+import xml2js from "xml2js";
+import ModelInfo, {
+  ModelInfoXml,
+} from "../../../src/model/modelInfo/ModelInfo";
+import SimpleElement from "../../../src/model/modelInfo/SimpleElement";
+import TypeInfo from "../../../src/model/modelInfo/TypeInfo";
+import { ElementXml } from "../../../src/model/modelInfo/ElementFactory";
+import ChoiceElement from "../../../src/model/modelInfo/ChoiceElement";
+import ListElement from "../../../src/model/modelInfo/ListElement";
+
+describe("ModelInfo", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)